dapps/marketplace/src/pages/listings/_EarnTokensCard.js

```diff
@@ -30,41 +33,91 @@ const EarnTokensCard = ({ identity }) => {
       </h5>
     </div>
   )
+})
+
+const BuyListingsForRewards = withRouter(({ history }) => {
+  return (
+    <div className="listing-card earn-tokens-card">
+      <div
+        className="main-pic"
+        onClick={() => {
+          history.push({
```

Comment:
We can't use the `Redirect` component here? That component handles scrollToTop too

Reply:
Will update this.